src/flb_utils.c

```diff
@@ -706,7 +706,10 @@ int flb_utils_write_str(char *buf, int *off, size_t size,
             encoded_to_buf(p, tmp, len);
             p += len;
         }
-        else if (c >= 0x80 && c <= 0xFFFF) {
+#if defined(__powerpc64__)
```

Comment:
I would like to understand this part before this is merged so I'll mark this review as asking for changes just so no one merges it by mistake.

Could you explain the logic behind this change and why is it for ppc64le support?

Reply:
The values being received for power were in a different range, and needed the logic from the _else if (c > 0xFFFF) {_ part of the code. There were two failing tests, which passed after this change. Thats the best I know.

Reply:
Could you please explain why would that happen? at first (when I hadn't read your PR message) I thought it could've been due to the endianness difference but considering that supporting little endian is the point of the PR that doesn't make sense (obviously).

So I think we need to properly understand why those cases were failing, what the difference with x86 or arm is and what the correct fix is (even if in the end it's just disabling the block with proper understanding of the reason why (and a comment)).